keyboards/pteron56/keymaps/brant/config.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+
```

Comment:
This is the [default behaviour](https://github.com/qmk/qmk_firmware/blob/develop/quantum/command.h#L37).
```suggestion
```